tests/e2e/plugins/module-setup-tagmanager.php

```diff
@@ -0,0 +1,104 @@
+<?php
+/**
+ * Plugin Name: E2E Tests Module Setup TagManager API Mock
+ * Plugin URI:  https://github.com/google/site-kit-wp
+ * Description: Utility plugin for mocking TagManager Setup API requests during E2E tests.
+ * Author:      Google
+ * Author URI:  https://opensource.google.com
+ *
+ * @package   Google\Site_Kit
+ * @copyright 2019 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://sitekit.withgoogle.com
+ */
+
+namespace Google\Site_Kit\Tests\E2E\Modules\TagManager;
+
+use Google\Site_Kit\Core\REST_API\REST_Routes;
+
+const ACCOUNT_ID_A = '100';
+const ACCOUNT_ID_B = '101';
+
+const PUBLIC_ID_X = 'GTM-ABCXYZ';
+const PUBLIC_ID_Y = 'GTM-BCDWXY';
+
+const CONTAINER_ID_X = '200';
+const CONTAINER_ID_Y = '201';
+
+function filter_by_account_id( $items, $account_id ) {
+	return array_values(
+		array_filter(
+			$items,
+			function ( $item ) use ( $account_id ) {
+				return $item['accountId'] === $account_id;
+			}
+		)
+	);
+}
+
+//function filter_by_public_id( $items, $public_id ) {
```

Comment:
What's with this commented out code here? Might wanna remove it if not needed.

Reply:
Hmmm, I would think PHPCS should have caught that. I guess it's not running on E2E code?

Reply:
Actually, it looks like it is not running for any test code... which is a bit  odd.